.github/workflows/apm-integrations.yml

```diff
@@ -6,13 +6,22 @@ on:
     branches: [master]
   schedule:
     - cron: 0 4 * * *
+  workflow_dispatch:
+    inputs:
+      nightly-version:
+        description: 'Nightly node version to run'
+        required: false
+        default: '24'
```

Comment:
In the current setup, the actual default is provided  by the underlying `node` action, so I would keep this as an empty string (so probably leave it undefined as that's probably the default for strings).